app/api/event_copyright.py

```diff
@@ -71,20 +73,20 @@ def before_create_object(self, data, view_kwargs):
 
 class EventCopyrightDetail(ResourceDetail):
 
-    def before_patch(self, args, kwargs):
+    def before_patch(self, args, kwargs, data):
```

Comment:
Because without this, it would be (or rather was) giving an error. Because we are sending a data argument in the PATCH, but in this function, we didn't have any  data parameter. So this is necessary I think.